Dockerfile

```diff
@@ -0,0 +1,43 @@
+#################################################################################
+#  Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
```

Comment:
The workflow for building the image is still missing right?
When adding the workflow could you please make sure to build multi platform? https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-08

One way would be using QEMU, like for instance PURIS does https://github.com/eclipse-tractusx/puris/blob/main/.github/workflows/build-image-backend.yml#L64

Reply:
Yes, a workflow to build and publish docker image is not added yet.
Sure, will take a ref for a mentioned link.